woocommerce/woocommerce
# [E2E test coverage]: Enable block product editor

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR introduces E2E tests to ensure that enabling the new experience works as expected.
It also removes the tests for the js product editor [new-product-editor.spec.js](https://github.com/woocommerce/woocommerce/pull/39463/files#diff-a0a31b4e3a608524704053535f1dfbb8d4c4d7e1bbca1219c66144d980c32808).

I wonder if we should remove [this plugin](https://github.com/woocommerce/woocommerce-experimental-enable-new-product-editor) in a follow-up PR after merging this. The plugin enables the (old) new experience.

Closes #39442.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Execute the standard commands to launch the local E2E environment.
2. Make sure the test is not flaky by running multiple instances of it repeatedly. You can do so through this command:
    ```bash
    cd plugins/woocommerce
    pnpm test:e2e-pw enable-block-product-editor.spec.js --retries=0 --workers=4 --repeat-each=8
    ```
4. All instances of this test should pass, which indicates it isn't flaky.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
